test/beta_pyrsh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import paramiko
+import argparse
+import sys
+from contextlib import contextmanager
+import time
+
+import subprocess
+import telnetlib
+import os
+
+class ssh:
```

Comment:
@GitSergeyGit the class name is wrong, it does not reflect the reality. It is not only about ssh right?

Reply:
Also this clss does not respec single responsibility prinsiple - it is a container for all the shells, it is not really maintainable and error prone